longevity_operator_multi_tenant_test.py

```diff
@@ -28,10 +28,11 @@ def _run_test_on_one_tenant(tenant):
 
         self.log.info("Starting tests worker threads")
 
-        self.log.info("Clusters count: %s", self.k8s_cluster.tenants_number)
+        tenants_number = self.k8s_clusters[0].tenants_number
```

Comment:
this temporary until we'll want multidc + multitenant setup ? right ?

Reply:
No, it is final. The `tenants_number` is the same all the time. One tenant or more, it doesn't matter, all spread over all the K8S clusters we use.